tutorials/nservicebus-101/lesson-1/tutorial.md

```diff
@@ -0,0 +1,144 @@
+
+
+#### Installers
+
+snippet:EnableInstallers
+
+This setting instructs the NServiceBus endpoint to run its installers on startup. Installers are used to set up anything the endpoint requires to run. The most common example is creating necessary queues, such as the endpoint's input queue where it will receive messages.
+
+
+### Starting up
+
+At the end of the `AsyncMain` method, after the configuration code, add the following code which will start up the endpoint, keep it running until we press the Enter key, and then shut it down.
+
+snippet:Startup
+
+When the endpoint starts, the `EndpointConfiguration` information is locked down from further changes, and the settings it contains are used to initialize the endpoint.
+
+When you run the endpoint for the first time, the endpoint will:
+
+ * Display its logging information, which is written to a file as well as the console, so if you miss something on the console, you can go back and look at the file. NServiceBus also logs to multiple levels, so you can [change the log level](/nservicebus/logging/) from `INFO` to log level `DEBUG` in order to get more information about what's going on.
+ * Display the [status of your license](/nservicebus/licensing/).
+ * Attempt to add the current user to the "Performance Monitor Users" group so that it can write [performance counters](/nservicebus/operations/performance-counters.md) to track its health and progress. If it is not able to do so (usually because the process is not running with elevated privileges) then it will show you how to do it manually from an admin console.
+ * Warn you that the [queues it created have too many permissions](/nservicebus/msmq/operations-scripting.md#create-queues-default-permissions). This makes it easier for development, but in a production scenario these queues should be created with the minimum required privileges, and these warnings will serve as reminders to do that.
+
+
+## Summary
+
+In this lesson we set up the prerequisites for NServiceBus and created a simple messaging endpoint to make sure it works. In the next lesson, we'll define a message, a message handler, and then send the message and watch it get processed.
+
+Before moving on, you might want to check your code against the completed solution (below) to see if there's anything you may have missed.
+
+When you're ready, move on to [**Lesson 2: Sending a command**](../lesson-2/).
```

Comment:
considering all the text above is conversation tone i think u have achieved that already. lets put that text in the template instead